__tests__/app.test.js

```diff
@@ -178,6 +179,27 @@ describe("/api", () => {
           //       ).toBe(true);
           //     });
           // });
+          describe("POST/api/reviews/review_id/comments", () => {
```

Comment:
There are no tests? 

 - 400 - invalid id
 - 400 - missing required fields in sent in comment 
 - 404 - non-existent id


 There are possible others but these three need to be implemented.